spec/requests/cats_spec.rb

```diff
@@ -40,8 +40,28 @@
     cat = Cat.first
     p "cat:", cat
     expect(cat.name).to eq('Steven')
-    expect(cat.enjoys).to eq('climbing')
+    expect(cat.enjoys).to eq('climbing around')
 
     end
   end
+
+    it "will not create a cat missing its name" do
+      cat_params = {
+       cat: {
+        age: 5,
+        enjoys: 'climbing',
+        image: 'https://upload.wikimedia.org/wikipedia/commons/1/1f/Cat_climbing_tree%2C_Uchimaki_Park.jpg'
+      }
+    }
```

Comment:
Since the test is to check for the name validation error message, make sure that the other keys are not generating errors too. Recommend modifying the value for enjoys key so it does not trigger the minimum length validation.

Reply:
Okay i see what you're saying, we ran into this problem initially but resolved it and changed the top two to 'climbing around' but we must have missed this one, thank you!